mwdb/web/src/components/UserProfile.js

```diff
@@ -84,47 +113,54 @@ class UserProfile extends Component {
                             <td>Last password set</td>
                             <td><DateString date={this.state.profile.set_password_on}/></td>
                         </tr>
+                        {this.capabilities.length > 0 &&
+                        <tr>
+                            <td>
+                                Capabilities
+                            </td>
+                            <td style={{padding: 22}}>
```

Comment:
Avoid adding values without the unit. It will fallback to pixels but we should not rely on that.

That one looks bad because `<ul>` is missing. For `ul` element, use CSS below to remove the left padding (preferred to make a class for in-table ul's and put in in CSS file instead of inline):
```
padding: 0;
list-style-position: inside;
```